src/integrations/GA4/browser.js

```diff
@@ -47,7 +47,7 @@ export default class GA4 {
     if (this.analytics.userId) {
       gtagParameterObject.user_id = this.analytics.userId;
     }
-
+    gtagParameterObject.cookie_prefix = 'rs';
```

Comment:
Have we verified this will stop the conflict between ga4 and ga ua?

Reply:
yes @ItsSudip